src/structures/Guild.js

```diff
@@ -959,6 +970,49 @@ class Guild extends Base {
       .then(data => GuildAuditLogs.build(this, data));
   }
 
+  /**
+   * Data for a field in Membership Screening
+   * @typedef {Object} GuildMembershipScreeningField
+   * @property {MembershipScreeningType} fieldType The type of the field
+   * @property {string} label The title of the field
+   * @property {string[]} [values] The list of values in the field
+   * @property {boolean} required Whether the user has to fill out this field
+   */
+
+  /**
+   * Data for the Guild Membership Screening object
+   * @typedef {Object} GuildMembershipScreening
+   * @property {boolean} enabled Whether membership screening is enabled
+   * @property {string} description The server description shown in the membership screening form
+   * @property {GuildMembershipScreeningField[]} formFields The steps in the membership screening form
+   */
+
+  /**
+   * Fetches the guild Membership Screening data.
+   * @returns {Promise<GuildMembershipScreening>}
+   * @example
+   * // Fetches the guild membership screening options
+   * guild.fetchMembershipScreening()
+   *   .then(memberScreen => console.log(`Membership Screening is ${memberScreen.enabled ? 'enabled' : 'disabled'}`))
+   *   .catch(console.error);
+   */
+  async fetchMembershipScreening() {
+    if (!this.features.includes('COMMUNITY')) {
+      throw new Error('COMMUNITY');
+    }
```

Comment:
I don't see this in https://github.com/discordjs/discord.js/blob/master/src/errors/Messages.js, and it doesn't seem to be added in this PR?

Reply:
Good point, I hadn't realized that Error's were extended when I wrote this, am aware now, will fix in next push